config/owasp/suppressions.xml

```diff
@@ -4,6 +4,7 @@
     <cve>CVE-2018-1258</cve>
     <cve>CVE-2021-25122</cve>
     <cve>CVE-2021-25329</cve>
+    <cve>CVE-2021-21295</cve>
```

Comment:
this CVE has just been resolved by https://github.com/hmcts/em-hrs-api/pull/82/files

Reply:
I've pulled in the CVE resolution